torchao/sparsity/sparse_api.py

```diff
@@ -72,8 +72,8 @@ def filter_fn(module: nn.Module, fqn: str) -> bool:
         m = sparsify_(m, semi_sparse_weight(), filter_fn)
 
         # for int8 dynamic quantization + 2:4 sparsity
-        from torchao.dtypes import SemiSparseLayoutType
-        m = quantize_(m, int8_dynamic_activation_int8_weight(layout_type=SemiSparseLayoutType), filter_fn)
```

Comment:
same for this, also please write a bc-breaking note for `int4_weight_only` and `int8_dynamic_activation_int8_weight` in the Summary